jellyfin_kodi/helper/utils.py

```diff
@@ -261,6 +261,23 @@ def validate(path):
     return True
 
 
+def validate_bluray_dir(path):
+
+    ''' Verify if path/BDMV/ is accessible.
+    '''
+
+    path = path + '/BDMV/'
+
+    path = path if os.path.supports_unicode_filenames else path
```

Comment:
Doesn't this evaluate to `path` regardless of whether it supports unicode or not?

Reply:
Yes, you are right.
This line is just copied from **def validate(path)** in v0.5.1 and it works , so I didn't put much attention on it. I saw it removed in current version.